### ISSUE TITLE
Unit of avg loss

### ISSUE THREAD

**Issue by BS-98** [AlexeyAB/darknet#6659]
Hi,

could you tell me what is the unit of avg loss during training?

Sometimes I got 2600 avg loss, sometimes 300 and sometimes 3. I do not really understand this. Is it a [%] or something else?

**Comment by johnny-mueller** [comment]
Can you create a screenshot of it and post it here for a better understanding?

Which dataset are you working with?

**Comment by BS-98** [comment]
![chart_yolo-obj](https://user-images.githubusercontent.com/66216239/93181460-c7c49580-f738-11ea-9ab9-ef34282f3cd2.png)

I ended with avg loss ~3.9 but at the first iterations I was getting avg loss about 300. So, I do not really know what is the unit of this loss, how exactly is calculated. I work with aerial dataset on which I try to detecet cars. On above chart I put 416x416 images without resize but in other case I put 800x800 images with resize to 416x416 and avg loss was much higher.

**Comment by johnny-mueller** [comment]
Okey, so I don't know which model you are currently training but I'll just take yolov3 as my starting point. 

To go into the loss briefly, a very shortened incomplete general explanation:
The loss that is calculated by the loss function, also called cost function, is there to impose a value on errors. The bigger the error or the bigger the deviation from the ground truth, the bigger the loss.

Yolo has three detection layers that are responsible for calculating the loss from three different scales. These three losses are then summed for backpropagation.

I can only recommend you to look at the loss calculation from the original yolo paper. This should at least make some things clearer to you.

Also, I can only recommend that you do not scale the images down before training the network. This has been mentioned by AlexeyAB himself in several issues.

**Comment by BS-98** [comment]
I have to rescale as I have only 6GB GPU. I use yolov4-mish-416.cfg file and proper weights as describe in model zoo. 

As for the losses. There are three losses calculated for classification and three for regression? And on the chart avg loss is just average of them?
